### PR TITLE
Refactor of integration documents hierarchy

### PR DESCRIPTION
**Pull request by ryankscott**
## Test plan
* Navigate to [docs page](http://localhost:5080/integration/browser_extension)
* Go through each subsequent page and ensure links are not broken and images render

### CODE REVIEW

File: doc/integration/browser_extension/how-tos/google_workspace.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Users will need to have valid accounts and be signed in to your Sourcegraph inst
 
 Users may also need to grant additional permissions to the Sourcegraph browser extension in order for it to provide code navigation on your code host. To check if Sourcegraph is properly configured, navigate to any page on your code host, and click the Sourcegraph icon in the browser toolbar. The popup will prompt you to grant additional permissions if needed:
 
-![](img/permissions.png)
```
Comment (philipp-spiess) [inline doc/integration/browser_extension/how-tos/google_workspace.md:36]: This image should probably have an alt text

File: doc/integration/browser_extension/how-tos/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+1. Click the Sourcegraph extension icon in the browser toolbar to open the settings page.
+    - Ensure that the Sourcegraph URL is correct. It must point to your own Sourcegraph instance to work on private code.
+    - Check whether any permissions must be granted. If so, the settings page will offer you to "grant the Sourcegraph browser extension additional permissions".
+1. On some code hosts, you need to be signed in (to the code host) to use the browser extension. Try signing in.
+
+## The *Enable Sourcegraph on this domain* option is not available
+
+In rare cases, Chrome can get into the state where the option to **Enable Sourcegraph on this domain** is not available when right-clicking on the extension icon. One fix we've observed is to toggle the site access from on, to off, then on again (see below).
+
+![Toggle site access for browser extension ](https://sourcegraphstatic.com/ToggleSiteAccess.gif)
```
Comment (philipp-spiess) [inline doc/integration/browser_extension/how-tos/troubleshooting.md:20]:
```suggestion
![Toggle site access for browser extension](https://sourcegraphstatic.com/ToggleSiteAccess.gif)
```

**Review by philipp-spiess — APPROVED** [review]
Deadpool LGTM

File: doc/integration/browser_extension/references/privacy.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Sourcegraph integrations will only connect to Sourcegraph.com as required to provide code navigation or other functionality on public code. As a result, no private code, private repository names, usernames, or any other specific data is sent to Sourcegraph.com.
+
+If connected to a **private, self-hosted Sourcegraph instance**, Sourcegraph integrations never send any logs, pings, usage statistics, or telemetry to Sourcegraph.com. They will send notifications of usage to that private Sourcegraph instance only. This allows the site admins to see usage statistics.
+
+If connected to the **public Sourcegraph.com instance**, Sourcegraph integrations will send notifications of usage on public repositories to Sourcegraph.com.  
+
+The browser extension also does not store sensitive data locally. The information stored is restricted to:
+
+- AnonymousUid
+-  Feature flags
```
Comment (abeatrix) [inline doc/integration/browser_extension/references/privacy.md:12]:
```suggestion
- Feature flags
```

File: doc/integration/index.md [hunk 4]
```diff
@@ -2,8 +2,7 @@
 
 Sourcegraph integrates with your other tools to help you search, navigate, and review your code.
 
-- [Browser extension](browser_extension.md): go-to-definitions and hovers in your code host and code reviews
-  - [Google Workspace (formerly known as G Suite) and Chrome Enterprise](google_workspace.md)
+- [Browser extension](./browser_extension/index.md): go-to-definitions and hovers in your code host and code reviews
```
Comment (abeatrix) [inline doc/integration/index.md:5]:
```suggestion
- [Browser extension](browser_extension/index.md): go-to-definitions and hovers in your code host and code reviews
```

I think you will get an error if you add `./` to the front but i'm not sure

**Review by abeatrix — APPROVED** [review]
lgtm! 
It looks like there is [a list of pages generated by buildkite](https://buildkite.com/sourcegraph/sourcegraph/builds/169978#0182eefc-6d5c-44d0-a2a6-34aaabaccd21) that needs updating after these changes 👀

**Pull request merged**